### ISSUE TITLE
White line at the right border

### ISSUE THREAD

**Issue by leonardo-gooten** [huynhsontung/Screenbox#453]
**Describe the bug**
As can be seen in the pictures, there is a white line at the right border. It happens on all videos I tested, downloaded or recorded ones. mkv and mp4 files. 
**How To Reproduce**
Just open the video

**Expected behavior**
not white lines

**Screenshots**
![image](https://github.com/user-attachments/assets/716ac733-9b50-420f-881a-99d6a450c326)
Can even see the mouse icon blocking the white line in one of the pics
![IMG_20240926_164410](https://github.com/user-attachments/assets/5c52d12b-3b74-45d5-aca7-47e7810f6191)
![IMG_20240926_164414](https://github.com/user-attachments/assets/f50f3c37-cc84-4c22-9060-df6edc1501a2)

**Environment**
 - OS:  Windows 11,
 - Device: Samsung Galaxy Book3 ultra
 - App Version v0.13.2.0

**Additional context**
None

**Comment by huynhsontung** [comment]
Thanks for reporting. Can you see the white line when the app is not in fullscreen? What is your display scaling percentage? Can you try changing your display scaling to any other values?

**Comment by ltlombardi** [comment]
Yes, the line is visible even without fullscreen.
The problem only happens if I'm in Light Mode on windows 11.
I confirm that the problem happens when scaling is in175%. If I change to 200 or to 150% the problem doesn't happen.

![image](https://github.com/user-attachments/assets/5812af4a-8c44-46cc-8314-2752778b5bfd)
![image](https://github.com/user-attachments/assets/c4959373-13fe-4ccc-8ad0-00112e040636)

**Comment by United600** [comment]
> Hi there, I think i have the same problem, and my scaling is 150%, and If I change it to 200% the problem doesn't happen. OS: Windows 11, Device: HUAWEI Mate Pad E GO 2022 Chip: Snapdragon (TM) 8cx Gen 3 App Version v0.14.1.0

Can you provide a clearer screenshot of the affected region? It's difficult to identify the issue, thanks.

**Comment by Minessential** [comment]
> > Hi there, I think i have the same problem, and my scaling is 150%, and If I change it to 200% the problem doesn't happen. OS: Windows 11, Device: HUAWEI Mate Pad E GO 2022 Chip: Snapdragon (TM) 8cx Gen 3 App Version v0.14.1.0
> 
> Can you provide a clearer screenshot of the affected region? It's difficult to identify the issue, thanks.


Notice the part inside the blue box, this frame should be pure black.
![image](https://github.com/user-attachments/assets/86f2eb50-4bc8-4b4c-bb45-a3b63153b1b9)

**Comment by United600** [comment]
Oh! I see. Are those "artifacts" visible in every single video?

**Comment by Minessential** [comment]
> Oh! I see. Are those "artifacts" visible in every single video?

yes, every video. and when i resize the windows, the "artifacts" will change position on the window.

in addition, the vlc player is have the same problem.
![image](https://github.com/user-attachments/assets/6f278f28-c3ba-4098-b9a2-48a905b41c6a)

**Comment by United600** [comment]
> yes, every video. and when i resize the windows, the "artifacts" will change position on the window.
>
> in addition, the vlc player is have the same problem.

Right, this is a separate issue. It might be a simple bug in the graphics driver, do you have the latest version?

Alternatively the app, along with LibVLC, ARM64 support is still far from flawless.

**Comment by Minessential** [comment]
> > yes, every video. and when i resize the windows, the "artifacts" will change position on the window.
> > in addition, the vlc player is have the same problem.
> 
> Right, this is a separate issue. It might be a simple bug in the graphics driver, do you have the latest version?
> 
> Alternatively the app, along with LibVLC, ARM64 support is still far from flawless.

yes, my graphics driver is the latest version.
